components/ResourceDetail/Masthead.vue

```diff
@@ -100,6 +100,10 @@ export default {
       return null;
     },
 
+    resourceSubtypeString() {
+      return this.resourceSubtype && this.resourceSubtype.length ? `${ this.resourceSubtype } -` : this.resourceSubtype;
```

Comment:
This can be shortened to `return this.resourceSubtype?.length ? '${ this.resourceSubtype } -' : ''`